docs/advanced_usage.md

```diff
@@ -221,13 +221,29 @@ $ notary remove example/collections delegation/path/target --roles=targets/relea
 
 ## Use delegations with content trust
 
-Docker Engine 1.10 supports the usage of the `targets/releases` delegation,
-other delegation roles will be supported in a future release. By default, Engine
-1.10 with content trust enabled will attempt to retrieve images signed by the
-`targets/releases` role if it exists; if it does not exist, the Engine  will
-fall back to retrieving images signed by the default `targets` role.
-
-To use the `targets/releases` role for pushing and pulling images with content trust, follow the steps above to add and publish the delegation role with notary. When adding the delegation, the `--all-paths` flag should be used. By default, pushing with Docker Content Trust will attempt to add to whichever delegations exist directly under targets (ex: `targets/releases`, but not `targets/nested/delegation`).
+Docker Engine 1.10 and above supports the usage of the `targets/releases`
+delegation as the canonical source of a trusted image tag, if it exists.
+
+When running `docker pull` with Docker Content Trust on Docker Engine 1.10,
+Docker will attempt to search the `targets/releases` role for the signed image tag,
+and will fall back to the default `targets` role if it does not exist.  Please note
+that when searching the default `targets` role, Docker 1.10 may pick up on other
+non-targets/releases delegation roles' signed images if they exist for this tag.
+In Docker 1.11, this behavior is removed such that all `docker pull` commands with
```

Comment:
Non-blocking nit:  maybe "removed" should be "changed", since the fallback behavior is unchanged, but the extra delegation role searching behavior is.